### PR TITLE
add doc for migration with data move

### PR DESCRIPTION
**Pull request by jmlrt**
Should fix https://github.com/elastic/ansible-elasticsearch/issues/581

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins retest this please

File: docs/multi-instance.md [hunk 1]
```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
 
 If you have single-instances hosts and want to upgrade from previous versions of the role:
 
+### Procedure with datas move
```
Comment (Conky5) [inline docs/multi-instance.md]: `datas` -> `data`, this is in a few places.

File: docs/multi-instance.md [hunk 2]
```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
...
+Example:
+```
+$ systemctl stop elasticsearch
+$ mv /etc/elasticsearch/${ES_INSTANCE_NAME}/* /etc/elasticsearch/ && rm -fr /etc/elasticsearch/${ES_INSTANCE_NAME}/
+mv: overwrite '/etc/elasticsearch/elasticsearch.keystore'? y
+$ mv /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/lib/elasticsearch/ && rm -fr /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+$ ls /var/lib/elasticsearch/
+nodes
+$ mv /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/log/elasticsearch/ && rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+$ rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
```
Comment (Conky5) [inline docs/multi-instance.md]: This `rm` happens in the previous line as well.

File: docs/multi-instance.md (hunk 1 above)
Comment (jmlrt) [reply docs/multi-instance.md]: @Conky5 fixed

File: docs/multi-instance.md (hunk 2 above)
Comment (jmlrt) [reply docs/multi-instance.md]: @Conky5 fixed

File: docs/multi-instance.md [hunk 3]
```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
...
+# mv /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/lib/elasticsearch/ && rm -fr /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# ls /var/lib/elasticsearch/
+nodes
+# mv /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/log/elasticsearch/ && rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# rm -fr /var/run/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+```
+
+3. Update playbook (remove `es_conf_dir`, `es_data_dirs`, `es_log_dir`, `es_pid_dir` and `es_instance_name` variables)
+
+4. Deploy ansible-role
```
Comment (Conky5) [inline docs/multi-instance.md]: Perhaps explicitly call out to use the new ansible role hre

Comment (jmlrt) [reply docs/multi-instance.md]: Ansible Galaxy command to install the new version of the role is in the example below: https://github.com/elastic/ansible-elasticsearch/pull/605/files#diff-05d70e203a6d81fe483f4556abe6750dR44

Is it what you expect?

Comment (Conky5) [reply docs/multi-instance.md]: Yes the example is what I expect, but there is not a step in the instructions that mentions updating the role or using the new role in step 4 here.

Comment (jmlrt) [reply docs/multi-instance.md]: fixed in https://github.com/elastic/ansible-elasticsearch/pull/605/commits/fb64bd7d1cd88e858614522d84e11dffddfc8041

**Pull request merged**